File: app/src/main/java/org/attendr/main/SplashActivity.java
package org.attendr.main;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import org.attendr.R;
import org.attendr.classes.ClassDetailsActivity;

public class SplashActivity extends AppCompatActivity {

	@Override
	protected void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		setContentView(R.layout.activity_splash);
        Intent i = new Intent(this, ClassDetailsActivity.class);
        startActivity(i);
	}
}

File: app/src/main/java/org/attendr/main/SplashActivity.kt
package org.attendr.main

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import org.attendr.R
import org.attendr.classes.ClassDetailsActivity

/**
 * @author Pauldg7@gmail.com (Paul Gillis)
 */
class SplashActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)
        val i = Intent(this, ClassDetailsActivity::class.java)
        startActivity(i)
    }
}